packages/atomic/src/components/recommendations/atomic-recs-list/atomic-recs-list.tsx

```diff
@@ -56,7 +63,8 @@ export class AtomicRecsList implements InitializableComponent<RecsBindings> {
   private nextNewResultTarget!: FocusTargetController;
 
   /**
-   * The desired layout to use when displaying results. Layouts affect how many results to display per row and how visually distinct they are from each other.
+   * How children results should be displayed. This does not affect the display of the list itself.
```

Comment:
Why `children` here? Doesn't this apply to all results?
```suggestion
   * The layout to apply when displaying results themselves. This does not affect the display of the surrounding list itself.
```